cmd/feed-ingress/main.go

```diff
@@ -62,6 +71,19 @@ func configureLogging() {
 	}
 }
 
+func configureHealthPort() {
+	http.HandleFunc("/health", checkHealth)
```

Comment:
Regarding nginx. Yes I think we should wait until we've called Start() on the LB at least. But not if it makes things too complicated